src/main/kotlin/io/github/orangain/prettyjsonlog/Extract.kt

```diff
@@ -108,11 +108,20 @@ fun extractLevel(node: JsonNode): Level? {
     }
 }
 
-private val messageKeys = listOf("message", "msg")
+private val messageKeys = listOf("message", "msg", "error.message")
```

Comment:
You added the `error.message` as a message key, but according to [the manual](https://logging.apache.org/log4j/2.x/manual/json-template-layout.html) and [the source code](https://github.com/apache/logging-log4j2/blob/rel/2.23.1/log4j-layout-template-json/src/main/resources/EcsLayout.json), it seems that the `message` key presents in the format.

This means that the `error.message` key will be used only when the `message` key does not present in the JSON log. Is such a thing usually the case? If not, I would like to keep the `messageKeys` as they are.

Reply:
im new to this format
i seen 1 or 2 version that dont use `message` on errors so my idea was that if `message` is absent and `error.message` is present then lets show any `error.message` 
it's just a suggestion, if you don't like it, let's withdraw it

Reply:
I see. You have actually seen such a log. Then let's leave the `error.message`.